views/js/review/plugins/content/item-answer/scss/item-answer.scss

```diff
@@ -161,9 +163,23 @@ $incorrectColor: blacken($error, .2);
         .navigable {
             position: relative;
             &:focus {
-                outline: $borderWidth dashed $textColor;
+                outline: $focusBorderWidth solid $info;
                 outline-offset: -$focusSpace;
             }
         }
+        li.active .navigable {
+            &:focus {
+                outline-offset: -$focusActiveSpace;
+            }
+        }
+        
     }
 }
+// for navigation
+.qti-itemBody {
```

Comment:
Since this is not related to the plugin, it should be put on the main layout.

Reply:
 After discussion with UX decided to remove focusing on the item.